RELEASE_NOTES.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
   - upgrade to geantyref 1.3.14
+  - removes the core dependency on the caffeine library. This now uses a simple FIFO cache for sql parser and sql statements. The old behavior can be restored by adding the `jdbi3-caffeine-cache` dependency and adding `jdbi.installPlugin(new CaffeineCachePlugin());`.
```

Comment:
fyi, this is now an LRU since you reorder after every access

Reply:
thanks for spotting this.